Dockerfile/Dockerfile

```diff
@@ -0,0 +1,30 @@
+FROM python:3.8-slim-buster
+
+RUN pip install --upgrade pip
+
+RUN apt-get update
+
+RUN apt-get install curl unzip --yes
+
+RUN apt-get update && apt-get install -y nodejs npm
+RUN curl -fsSL https://deb.nodesource.com/setup_16.x | bash - && apt-get install -y nodejs
```

Comment:
Change to version 18 the latest LTS release. For 16 the active support ended 3 months ago and security patches will only provided for the next 7 months.

Reply:
+1 The apt-get should already be enough